api/envoy/extensions/access_loggers/file/v3/file.proto

```diff
@@ -30,16 +33,26 @@ message FileAccessLog {
     // Access log :ref:`format string<config_access_log_format_strings>`.
     // Envoy supports :ref:`custom access log formats <config_access_log_format>` as well as a
     // :ref:`default format <config_access_log_default_format>`.
+    // Will be deprecated.
```

Comment:
Please actually deprecate all of these fields, and add a deprecated note to the version history.

Reply:
Done